lms/static/sass/discussion/_edx-pattern-library-loader.scss

```diff
@@ -0,0 +1,23 @@
+// Discussions - CSS application architecture
+// Version 1 styling (pre-Pattern Library)
+// ====================
+
+// Load the LTR version of the edX Pattern Library
+$pattern-library-path: '../../edx-pattern-library' !default;
+
+/* pattern library alters only - we don't want reset and normalization styles from it to leak into inline discussion */
+// TODO: make edx-pattern-library provide "bodyless" version without actual CSS rules (only SASS mixins, vars etc.)
+@import 'edx-pattern-library/pattern-library/sass/global/lib';                    // third-party libraries
+/* parts of pattern-library/sass/global/core */
+@import 'edx-pattern-library/pattern-library/sass/global/functions';
+@import 'edx-pattern-library/pattern-library/sass/global/settings';
+@import 'edx-pattern-library/pattern-library/sass/global/mixins';
+@import 'edx-pattern-library/pattern-library/sass/global/helpers';
+/* end parts of pattern-library/sass/global/core */
+@import 'edx-pattern-library/pattern-library/sass/patterns/alerts';               // Alerts styling
```

Comment:
@pomegranited unnecessary - the entire pattern lib was added to fix single error with alert messages.